docs/SETUP.md

```diff
@@ -40,26 +52,24 @@ $ helm version
 version.BuildInfo{Version:"v3.11.3", GitCommit:"323249351482b3bbfc9f5004f65d400aa70f9ae7", GitTreeState:"clean", GoVersion:"go1.20.3"}
 ```
 
-## Compile and install Kind
+## Install kind
 
-We can use `kind` to deploy a single-node k8s cluster on any development machine.
-Kind is preferable to others solutions like microk8s because it does not require sudo permissions.
-It does however require that the user be able to run docker containers (e.g., that the user is added to the docker group).
-Kind does not have GPU support, but there exists a [forked version](https://jacobtomlinson.dev/posts/2022/quick-hack-adding-gpu-support-to-kind/) (with minimal diff) enabling this.
-We start by compiling kind with GPU support from source:
 ```shell
-git clone -b gpu https://github.com/jacobtomlinson/kind
-cd kind
-make build
-mv bin/kind ~/bin/
+# Based on https://kind.sigs.k8s.io/docs/user/quick-start#installing-from-release-binaries
+# For AMD64 / x86_64
+[ $(uname -m) = x86_64 ] && curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.23.0/kind-linux-amd64
+# For ARM64
+[ $(uname -m) = aarch64 ] && curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.23.0/kind-linux-arm64
+chmod +x ./kind
+sudo mv ./kind /usr/local/bin/kind
```

Comment:
This would still not be possible on a machine without sudo without asking an admistriator to go through all these steps. I would either advocate:
1. We leave the binaries in `~/bin`
2. We move all the other ones to `/usr/local/bin` (e.g., helm etc).
At least this way we are consistent.

Reply:
Indeed. Updating this one...